src/services/game-logic/ColorsEnum.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Enum of chess piece colors.
+ * @readonly
+ * @enum {number}
+ */
+enum ColorsEnum {
```

Comment:
powinniśmy raczej unikać wrzucania nazw typów danych, samo Colors chyba wystarczy?

Reply:
osobiście wolę do enumów notację węgierską

Reply:
ale to chyba nie jest notacja węgierska? w sensie notacja węgierska odbywa się na zasadzie prefixów

Reply:
może i taka jest słownikowa definicja, ale w praktyce nie spotkałam się jeszcze z prefiksowaniem. Raczej z postfiksami